src/UI/Layouts/HUD/VON/VoNOverlay.layout

```diff
@@ -0,0 +1,14 @@
+FrameWidgetClass {
+ {
+  VerticalLayoutWidgetClass "{59389903A80C9C04}" {
+   Prefab "{59389903A80C9C04}"
+   {
+    VerticalLayoutWidgetClass "{00000000AB3C9068}" {
+     Prefab "{00000000AB3C9068}"
+     "Is Enabled" 0
+     "Is Visible" 0
```

Comment:
Disable both? Why not only "Is Visible"?

Reply:
Should only be not visible but enabled. Saving Layouts does not always really save it.